ql/src/java/org/apache/hadoop/hive/ql/stats/ColStatsProcessor.java

```diff
@@ -221,7 +222,7 @@ public int persistColumnStats(Hive db, Table tbl) throws HiveException, MetaExce
 
       start = System. currentTimeMillis();
       if (tbl != null && tbl.isNonNative() && tbl.getStorageHandler().canSetColStatistics(tbl)) {
-        boolean success = tbl.getStorageHandler().setColStatistics(tbl, colStats);
+        boolean success = tbl.getStorageHandler().setColStatistics(tbl, colStats, colStatDesc);
```

Comment:
colStatDesc redundant

Reply:
Yes, but we will need it for the next PR for individual columns .

Reply:
why don't we introduce it there?

Reply:
sure, removed.